src/main/java/org/candlepin/subscriptions/capacity/CapacityReconciliationController.java

```diff
@@ -93,7 +93,6 @@ public void reconcileCapacityForSubscription(Subscription subscription) {
 
     reconcileSubscriptionCapacities(subscription);
     reconcileSubscriptionProductIds(subscription);
-    subscriptionRepository.save(subscription);
```

Comment:
I'm not 100% sure, but it seems like there is an oversight in SubscriptionSyncController: https://github.com/RedHatInsights/rhsm-subscriptions/blob/b31011e9e90dab75df280b7fcaf9cc8066f02183/src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java#L236 might trigger a bug of some sort. (That line seems like it should probably be replaced with a call to reconcileCapacityForSubscription for the just-saved subscription in either branch of the conditional).

Reply:
I went ahead and adjusted that logic in a commit.